otx/cli/tools/eval.py

```diff
@@ -58,11 +59,6 @@ def parse_args():
         parser.add_argument("template")
     parser.add_argument("--data", required=False, default="./data.yaml")
     required = not os.path.exists("./data.yaml")
```

Comment:
Please, delete any mentions of hardcoded os-specific relatives paths

Reply:
Hmm, actually this kind of sentences are existed in many part.
How about eliminating all things in the separate PR?